test/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:12
+
+ENV password=notpassword1
+ENV dbpass=notpassword2
+ENV pswd=notpassword3
+ENV pass=notpassword4
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High <!-- 31e0e36a-b47e-4c44-af64-7ffd1807587e -->